dev-docs/workflows/attestationconfigapi.md

```diff
@@ -0,0 +1,17 @@
+# Attestation config API
+
+## Azure SEV-SNP
+The version numbers of SEV-SNP are updated as part of [e2e_verify](/.github/actions/e2e_verify/action.yml).
+Because the version numbers are not publicly posted by Azure, we observe the versions on Azure VMs and assume a global rollout after a threshold time.
+
+This estimate might make manual intervention necessary when a global rollout didn't happen.
+
+### Manually delete a version
+```
+COSIGN_PASSWORD=$PWD COSIGN_PRIVATE_KEY="$(cat $PATH_TO_KEY)" AWS_ACCESS_KEY_ID=$ID AWS_ACCESS_KEY=$KEY bazel run //internal/api/attestationconfigapi/cli delete -- --version 2023-09-02-12-52
+```
+
+### Manually upload a version
+```
+COSIGN_PASSWORD=$PWD COSIGN_PRIVATE_KEY="$(cat $PATH_TO_KEY)" AWS_ACCESS_KEY_ID=$ID AWS_ACCESS_KEY=$KEY bazel run //internal/api/attestationconfigapi/cli -- --force --version 2023-09-02-12-52  --maa-claims-path "${path}"
```

Comment:
Usage of `$PWD` for the cosign password seems incorrect here.
Should be `$CPWD` to align with the CLI tool description (though I would prefer something like `CPW` (as in **c**osign **p**ass**w**ord) to avoid confusion with the env variable `PWD`)

Reply:
$PWD is only a placeholder to be substituted by the user. It doesn't make sense to replace one env variable with another right?

Reply:
PWD is commonly used by Linux systems to return the current working directory.
Reusing it here seems not ideal to me